src/pywink/devices/standard.py

```diff
@@ -289,6 +304,9 @@ def _update_state_from_response(self, response_json):
                 outlet['last_reading']['connection'] = power_strip_reading.get('connection')
                 self.json_state = outlet
 
+    def pubnub_update(self, json_response):
```

Comment:
I'm not sure I see the purpose of all of these ``` pubnub_update ``` definitions.  Why wouldn't the library user simply set ``` device.json_state ``` directly?